cli/common_test.go

```diff
@@ -63,6 +63,29 @@ func Test_Common_createRequest(t *testing.T) {
 				},
 			},
 		},
+
+		// This test ensures that loading a single unit from a directory with the
+		// slice expression "@1", "@foo" and "@5" results in the expected
+		// controller request.
+		{
+			Setup: []testFileSystemSetup{
+				{
+					FileName:    "dirname/dirname_unit@.service",
+					FileContent: []byte("some unit content"),
+					FilePerm:    os.FileMode(0644),
+				},
+			},
+			Input: []string{"dirname@1"},
+			Expected: controller.Request{
+				SliceIDs: []string{"1", "foo", "5"},
```

Comment:
The input should provide that. The fact that the test works anyway is bad. Found the bug. Fixed the test.